src/core_modules/capture-core/components/Pages/MainPage/WorkingListsCommon/actions/workingListsCommon.actions.js

```diff
@@ -75,65 +70,68 @@ export const addTemplateSuccess = (templateId: string, clientId: Object, data: O
 export const addTemplateError = (clientId: Object, data: Object) =>
     actionCreator(workingListsCommonActionTypes.TEMPLATE_ADD_ERROR)({ clientId, ...data });
 
-export const cleanSkipInitAddingTemplate = (template: Object, listId: string) =>
-    actionCreator(workingListsCommonActionTypes.TEMPLATE_ADD_SKIP_INIT_CLEAN)({ template, listId });
+export const cleanSkipInitAddingTemplate = (template: Object, storeId: string) =>
+    actionCreator(workingListsCommonActionTypes.TEMPLATE_ADD_SKIP_INIT_CLEAN)({ template, storeId });
 
-export const deleteTemplate = (template: Object, programId: string, listId: string) =>
-    actionCreator(workingListsCommonActionTypes.TEMPLATE_DELETE)({ template, programId, listId });
+export const deleteTemplate = (
+    template: Object,
+    programId: string,
+    { storeId, workingListsType }: { storeId: string, workingListsType: string}) =>
+    actionCreator(workingListsCommonActionTypes.TEMPLATE_DELETE)({ template, programId, storeId, workingListsType });
```

Comment:
I am thinking that perhaps passing all the arguments as object would be an alternative to this signature. 
Why did you choose to split them like so?

Reply:
Because I'm adding the last argument to the already existing argument list in the `useWorkingListsCommonStateManagement` file.